src/pageEditor/tabs/editTab/editorNodeLayout/usePipelineNodes.tsx

```diff
@@ -426,22 +437,32 @@ const usePipelineNodes = (): {
       };
     }
 
+    const subPipelineHeaderActive =
+      activeNodePreviewElementId === null
+        ? false
+        : subPipelines.some(
+            ({ path }) =>
+              activeNodePreviewElementId ===
+              getNodePreviewElementId(blockConfig, path)
+          );
+
     const restBrickNodeProps: Except<
       BrickNodeProps,
       keyof BrickNodeContentProps
     > = {
       onClickMoveUp,
       onClickMoveDown,
       onClick,
-      active: nodeIsActive,
+      active: subPipelineHeaderActive ? false : isNodeActive,
```

Comment:
```suggestion
      active: !subPipelineHeaderActive && isNodeActive,
```
ie: `active` should only be true when `subPipelineHeaderActive` is `false` and `isNodeActive` is `true`